runtime/src/bank.rs

```diff
@@ -4029,9 +4029,11 @@ impl Bank {
                         .minimum_balance(nonce::State::size()),
                     _ => 0,
                 };
-                if lamports + min_balance > account.lamports() {
-                    return Err(TransactionError::InsufficientFundsForFee);
-                }
+
```

Comment:
Might as well un-`pub` this method also? L4022

Reply:
Done!